test/compile_errors.zig

```diff
@@ -219,4 +219,22 @@ pub fn addCases(ctx: *Cases) !void {
             \\pub fn anytypeFunction(_: anytype) void {}
         );
     }
+
+    {
+        const case = ctx.obj("imported function call with runtime-known argument to comptime param", .{});
+
+        case.addError(
+            \\export fn foo() void {
+            \\    var runtime: u8 = 0;
+            \\    @import("other.zig").foo(runtime);
+            \\}
+        , &[_][]const u8{
+            ":3:30: error: runtime-known argument passed to comptime parameter",
+            ":1:21: note: declared comptime here",
+        });
+
+        case.addSourceFile("other.zig",
+            \\pub fn foo(comptime _: u8) void {}
+        );
+    }
```

Comment:
This is testing the exact same thing as `callComptimeBoolFunctionWithRuntimeBool` above.

Reply:
Huh, so it is, I totally missed that. @andrewrk, I'm leaving this as-is for you so you can see the CI is happy, but it'd be great if you could drop the last commit on merge